FidelityInternational/certbot-dns-qip
# certbot dns qip plugin

## First working version of QIP DNS plugin for certbot

This plugin attaches to certbot and allows for provisioning DNS-01 challenge via QIP REST API

We can use it against our internal ACME server and automatically provision TXT records with ACME challenge in QIP zone. 

## How to test

- check if the automated build ( github actions )is green  - it runs linter and pytest 
- follow instructions here to set up a dev env for certbot: https://github.com/FidelityInternational/certbot-dns-qip/tree/Certbot-dns-qip-plugin-poc#plugin-development
- create `creds.ini` file
- run certbot command to request a cert from dev ACME server and make sure the cert is generated

## Who

- not @StuWill or @combor
